scene/3d/skeleton.cpp

```diff
@@ -538,13 +538,20 @@ void Skeleton::unbind_child_node_from_bone(int p_bone, Node *p_node) {
 	ERR_FAIL_INDEX(p_bone, bones.size());
 
 	uint32_t id = p_node->get_instance_id();
-	bones.write[p_bone].nodes_bound.erase(id);
+
+	int index = bones[p_bone].nodes_bound.find(id);
+	if (index == -1) {
+		return; // Doesn't exist in the first place.
+	}
+
+	bones.write[p_bone].nodes_bound.remove(index);
```

Comment:
Are the `nodes_bound` required to be in order? If not, we can maybe use `remove_unordered()`.. :thinking:

Reply:
`Skeleton` itself does not require some specific order. But changing `nodes_bound` order affect the order of the returned array from `get_bound_child_nodes_to_bone()` and the `bound_children` property. So using `remove_unordered()` breaks compatibility I think.